client/src/components/common/Filter/style.css

```
@@ -32,7 +32,6 @@
 .label__swich-btn {
     display: flex;
```

You can put important here and delete duplicated class name 
```suggestion
    display: flex !important;
```

---

important is necessary because we used third part library

---

Done